walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -61,7 +58,7 @@ public void optionsForNoAttendees() {
     MeetingRequest request = new MeetingRequest(NO_ATTENDEES, DURATION_1_HOUR);
 
     Collection<TimeRange> actual = query.query(NO_EVENTS, request);
-    Collection<TimeRange> expected = Arrays.asList(TimeRange.WHOLE_DAY);
+    Collection<TimeRange> expected = Collections.singletonList(TimeRange.WHOLE_DAY);
```

Comment:
Why do we make this change? Wouldn't an `Arrays.asList` work?

Reply:
It would work i just thought it would be better since the arrays are going to be containing a single item

Reply:
Collection.singletonList is preferred over Arrays.asList, but ImmutableList.of() is preferable to both.

Reply:
Would you prefer i use immutable.listof or do you think it is fine as is ?

Reply:
The end result is the same since Collections.singletonList returns an immutable list so I'd recommend switching.

Reply:
Thanks! Make sure after the changes, it's consistent throughout the whole program.

Reply:
ping

Reply:
@TcgVanguardTroll